test/test_checks.py

```diff
@@ -34,6 +34,18 @@ def test_noqa_doubles(self):
         self.assertEqual(list(checker.run()), [])
 
 
+class DoublesMixedTestChecks(TestCase):
```

Comment:
It's ambiguous what mixed means. Maybe we should rename to `DoublesInlineSinglesMultilineTestChecks` and similar for other suites?